### PR TITLE
Im 108 change login button

### PR DESCRIPTION
**Pull request by NamespaceValentine**

### CODE REVIEW

File: config/StoredProcedures/GetUserInfo.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+DROP PROCEDURE IF EXISTS GetUserInfo;
+
+DELIMITER $$
+CREATE PROCEDURE GetUserInfo(IN _Username varchar(256))
+BEGIN
+
+SELECT FirstName, LastName, Email FROM Users WHERE username=_Username;
+
+END $$
+DELIMITER ;
```
Comment (TimSwart) [inline config/StoredProcedures/GetUserInfo.sql]: This is fairly similar to GetUserByUsername. Do we need both?

Comment (NamespaceValentine) [inline config/StoredProcedures/GetUserInfo.sql]: We dont, you are right, let me fix that

File: routes/getUserInfo.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            res.send("error: " + error);
+        }
+        else {
+            Q.fcall(db.beginTransaction())
+                .then(db.query("USE " + db.databaseName))
+                .then(db.query("CALL GetUserByUsername ('"+val.username+"')"))
+                .then(function(rows, columns){
+                    var response = rows[0][0][0];
+                    res.send({
+                        FirstName: response.FirstName,
```
Comment (TimSwart) [inline routes/getUserInfo.js:21]: I would've thought you'd have to do a parse here before accessing the response fields. apparently not though.

Comment (NamespaceValentine) [inline routes/getUserInfo.js:21]: That is very weird. But yeah it works

**Pull request merged**